core/templates/dev/head/css/oppia.css

```diff
@@ -3187,6 +3187,20 @@ md-card.preview-conversation-skin-supplemental-card {
   color: blue;
 }
 
+.tooltip.right {
```

Comment:
This is going to affect every single tooltip in Oppia. I don't think you want that. Can you scope the CSS selector more specifically so that it only affects tooltips in the summary tile?